board/opentrons/ot2/rootfs-overlay/etc/nginx/nginx.conf

```diff
@@ -26,10 +26,12 @@ http {
         # longest prefix, so order shouldn't matter but length of path does
         location / {
             proxy_http_version       1.1;
-            proxy_set_header         Upgrade $http_upgrade;
-            proxy_set_header         Connection "upgrade";
             proxy_read_timeout       1h;
             proxy_pass               http://unix:/run/aiohttp.sock;
+
+            # Proxying these hop-by-hop headers is necessary for WebSockets.
+            proxy_set_header         Upgrade $http_upgrade;
+            proxy_set_header         Connection "upgrade";
```

Comment:
I thought we might want to delete these from the config because `/` is no longer a WebSocket endpoint.

But after reading this config more closely, it looks like this enables WebSockets for `/` *and all endpoints under `/`, recursively.* So even if we change our WebSocket endpoint from `/` to `/foo/subscribe` or whatever, this will help us. So I think these should be kept in the config as-is, for now.

Reply:
Looking at this again, I'm less confident in this. [This Server Fault answer](https://serverfault.com/a/408309/207333) suggests it *won't* apply recursively? Unless nginx treats the `/` location specially or something?

Leaving this alone for now.